### PR TITLE
Feature/32295 flask saml sso configuration

### PR DESCRIPTION
**Pull request by carlbordum**
https://redmine.magenta-aps.dk/issues/32295

**Please ensure the following is true:**

- [x] The application can be built and installed without errors
- [x] The feature/bugfix has been tested manually (if manually testable)
- [ ] Tests have been written/updated for my change
- [x] The documentation has been updated  (in code comments, default-settings is used by sphinx)
    - [x] The documentation builds without warnings or errors
- [ ] Release notes have been updated
- [x] The corresponding Redmine ticket has been set to `Needs review`
    - [ ] The issue has a link to this PR

**If applicable:**

- [ ] Changes have been made to the UI
    - [ ] Screenshots of relevant changes to UI added to PR
- [ ] The API/data model has been changed/expanded
    - [ ] Redmine tickets have been created to update dependent systems
- [x] The feature introduces changes relevant to the deployment process
    - [x] Redmine tickets have been created to update deployment process
- [ ] This change affects the development workflow.
    - [ ] I have notified other developers by email or chat.

### CODE REVIEW

File: backend/mora/default-settings.toml [hunk 1]
```diff
@@ -9,6 +9,7 @@
 # DO NOT CHANGE. You can configure OS2MO in the mo-dev-settings.toml file.
 
 
+session_cookie_name = "session"
```
Comment (cmoesgaard) [inline backend/mora/default-settings.toml]: This key should be under `[session]`

**Comment by carlbordum** [comment]
> There is some auth documentation including a minimal configuration example in `dev/auth.rst` (on the newest `development`). Please update this with up config values.

please re-review;

File: docs/dev/auth.rst [hunk 2]
```diff
@@ -44,6 +47,9 @@ The following additional configuration entries exist for auth in OS2MO.
 * ``"SAML_USERNAME_ATTR"``: If username is not read from NameID, the username
   will be read from an attribute with this name.
 
+All configuration of OS2MO is done as described in :ref:`settings`. The
+relevant section for authentication is called ``[saml_sso]``.
+
 Minimal example
```
Comment (cmoesgaard) [inline docs/dev/auth.rst]: Please update this section with new config values.

Comment (carlbordum) [reply docs/dev/auth.rst]: done

**Pull request merged**